bokeh/settings.py

```diff
@@ -608,8 +608,15 @@ def dev(self):
     A password to decrypt the SSL keyfile, if necessary.
     """)
 
-    strict = PrioritizedSetting("strict", "BOKEH_STRICT", convert=convert_bool, help="""
-    Whether validation checks should be strict (i.e. raise errors).
+    validation_level = PrioritizedSetting("validation_level", "BOKEH_VALIDATION_LEVEL", default="none",help="""
```

Comment:
We will want to make sure to give suitable error message if bad values are passed. This setting should model itself after the log level setting. See the `convert_logging` function here:

https://github.com/bokeh/bokeh/blob/bee3da0fbd4ec8b59b49f449ade9f7282481bb99/bokeh/settings.py#L212